src/coreclr/System.Private.CoreLib/src/ILLink/ILLinkTrim.xml

```diff
@@ -5,10 +5,6 @@
       <method name="_RegisterFrozenSegment" />
       <method name="_UnregisterFrozenSegment" />
     </type>
-    <!-- Accessed via private reflection by an external tracing controller. -->
-    <type fullname="System.Diagnostics.Tracing.EventPipe*" />
-    <!-- The private Event methods are accessed by private reflection in the base EventSource class. -->
-    <type fullname="System.Diagnostics.Tracing.NativeRuntimeEventSource" />
```

Comment:
Just tested this on a test branch with my changes and I think unrooting this will require more work for us... Could you change this to only unroot the `EventPipe` type for now?